src/ImageReconstruction.jl

```diff
@@ -36,20 +26,20 @@ function radon(image::AbstractMatrix, θ::AbstractRange, t::AbstractRange)
 
             (a < 1 || a > length(t)) && continue
             α = abs(t′ - t[a])
-            P[a, k] += (1 - α) * image[i, j]
+            P[a, k] += (1 - α) * I[i, j]
```

Comment:
Maybe this is a habit based on older versions of Julia's compiler, but I don't always trust Julia/LLVM's [common subexpression elimination](https://en.wikipedia.org/wiki/Common_subexpression_elimination), so I would assign `I[i, j]` to a temporary variable so you only have to access the array once. But I'm not certain this is still helpful.